.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,25 @@
+Fixes: #<issue>
+
+## Description
+
+Please, detail the root cause of the bug and how this PR fixes it.
+
+## PR review check list
+
+Quality check list that cannot be automatically verified.
+
+- [ ] Safety
+    - [ ] The changes respect the necessary conditions for safety
```

Comment:
What are those?

Reply:
What we have documented so far in the specs

Reply:
Can you then perhaps add a link to the spec here?

Reply:
done, however it's a bit of a long line

Reply:
Thanks. I would have asked to put it in actual markdown but then I realized that this will probably be rendered into plain text as it will appear in the user's textbox as soon as they open the PR so should be fine.